modules/gapi/src/backends/common/gcompoundkernel.cpp

```diff
@@ -16,7 +16,7 @@
 cv::detail::GCompoundContext::GCompoundContext(const cv::GArgs& in_args)
 {
     m_args.resize(in_args.size());
-    for (const auto& it : ade::util::indexed(in_args))
+    for (const auto it : ade::util::indexed(in_args))
```

Comment:
out of curiosity: y not `auto && it` ?

Reply:
details about that approach: https://stackoverflow.com/questions/13130708/what-is-the-advantage-of-using-forwarding-references-in-range-based-for-loops

Reply:
@anton-potapov ?

Reply:
In this context both versions are equal and correct, however "plain copy" version put less stress on the reader  (IMHO) more explicitly describing the intent